crates/sui-core/src/execution_cache.rs

```diff
@@ -455,6 +474,17 @@ pub trait ObjectCacheRead: Send + Sync {
         }
     }
 
+    fn have_updated_config_in_epoch(
+        &self,
+        object_id: &ObjectID,
+        epoch_id: EpochId,
+    ) -> SuiResult<bool> {
+        match self.get_marker_value(object_id, SequenceNumber::new(), epoch_id)? {
+            Some(MarkerValue::ConfigUpdate(_)) => Ok(true),
+            _ => Ok(false),
+        }
```

Comment:
Should we just call `get_last_config_object_change_sequence_number` with `is_some()`?